cmd/renterd/main.go

```diff
@@ -617,7 +617,7 @@ func main() {
 
 	// Finish worker setup.
 	if err := setupWorkerFn(context.Background()); err != nil {
-		logger.Fatal("failed to setup worker: " + err.Error())
+		logger.Warn("failed to setup worker: " + err.Error())
```

Comment:
Are we sure we want that? We already force users to provide the service address so it might be good to abort in case the address was wrong and the worker isn't reachable.

Reply:
This is debug leftover, thanks for catching that.